release-team/role-handbooks/enhancements/README.md

```diff
@@ -698,6 +698,51 @@ If there is continued unresponsiveness on issues, remove them from the milestone
 
 Exception process is outlined [here](https://github.com/kubernetes/sig-release/blob/master/releases/EXCEPTIONS.md)
 
+#### Exceptions.yaml File Format
+
+When tracking exceptions for a release, create an `exceptions.yaml` file in the release directory (e.g., `releases/release-1.35/exceptions.yaml`). Here's the expected format:
+
+```yaml
+# Exception requests in v1.35
+# Google Group: https://groups.google.com/g/kubernetes-sig-release
+# Release Team Lead: [Name] ([@github-handle](https://github.com/github-handle))
+
+# Enhancements Freeze Exceptions requested in v1.35
+
+enhancementFreeze:
+
+- name: "Feature Name"
+  issue: 1234
+  date_requested: 2025-01-15
+  date_reviewed: 2025-01-18
+  thread: https://groups.google.com/g/kubernetes-sig-release/c/thread-id
+  pull_requests:
+    - https://github.com/kubernetes/enhancements/pull/5678
+  status: "approved"  # or "rejected"
+
+# Code Freeze Exceptions requested in v1.35
+
+codeFreeze:
+
+- name: "Another Feature Name"
+  issue: 5678
+  date_requested: 2025-02-15
+  date_reviewed: 2025-02-18
+  thread: https://groups.google.com/g/kubernetes-sig-release/c/thread-id
+  pull_requests:
+    - https://github.com/kubernetes/kubernetes/pull/123456
+  status: "approved"  # or "rejected"
+```
+
+**Field Descriptions:**
+- `name`: Human-readable name of the enhancement
+- `issue`: Enhancement issue number in kubernetes/enhancements
+- `date_requested`: Date when the exception was requested (YYYY-MM-DD format)
+- `date_reviewed`: Date when the exception was reviewed by the release team (YYYY-MM-DD format)
```

Comment:
```suggestion
- `date_reviewed`: Date when the exception was formally approved/rejected by the release team (YYYY-MM-DD format)
```
nit: maybe make this wording more explicit, as "date reviewed" could be interpreted either as the date we started discussing the exception request, or the date we communicated the final decision

looks good to me otherwise, will let @rayandas give the final approval!

Reply:
Makes sense Jenny. Looks good to me.